src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.cs

```diff
@@ -20,11 +20,6 @@ public DriveInfo(string driveName)
             _name = NormalizeDriveName(driveName);
         }
 
-        private DriveInfo(SerializationInfo info, StreamingContext context)
-        {
-            throw new PlatformNotSupportedException();
-        }
```

Comment:
Isn't this used via reflection for deserialization?

cc: @ViktorHofer... am I wrong, or are we missing tests?

Reply:
> We have been deleting these private serialization constructors as part serialization cleanups

I'm not following @jkotas: those PRs didn't delete these ctors, it made them throw PNSE, e.g. https://github.com/dotnet/corefx/commit/a82f54bd51e22ec3eb14eb5d8f572f022becb8e3#diff-0de900803b15ed30c60fd6dd12c43013R31

Reply:
Ok. That makes sense.